reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -328,8 +330,19 @@ class StaticHtmlReporter : Reporter() {
                         +css
                     }
                 }
+                link(
+                    rel = "stylesheet",
+                    href = "//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/default.min.css"
+                )
+                script(src = "//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/highlight.min.js") {}
```

Comment:
`https:` is missing here. Also, are the empty curly braces really required?

Reply:
I took this from the docs, but it turns out the protocol is not added when loading the report from a local file, so added `https:`.
Yes, the braces are required, there is no default value for the parameter.